### ISSUE TITLE
Example seems to generate null instead of date.between

### ISSUE THREAD

**Issue by JoseGascon** [Cambalab/fake-data-generator#74]
**Describe the bug**
Example seems to generate null instead of date.between:
```
"publicationDate": {
        "type": "faker",
        "value": "date.between",
        "options": ["2019-01-02", "2019-12-29"]
```

**To Reproduce**
1. Execute the example  provided by fake-data-generator
2. The field "publicationDate" is always null

**Expected behavior**
dates should be generated randomly between both dates

**Comment by sgobotta** [comment]
Thanks for letting me know, @JoseGascon !

Looks like we've been giving the same treatment to Object and Array type options parameter from raw `.json` models.

I'll be addressing a PR with a fix.

**Comment by sgobotta** [comment]
Releases published under [v0.4.2](https://github.com/Cambalab/fake-data-generator/releases/tag/v0.4.2)
